11/streams/developer-guide/dsl-api.html

```diff
@@ -1484,14 +1484,6 @@ <h2><a class="toc-backref" href="#id7">Overview</a><a class="headerlink" href="#
                         Depending on the operands, joins are either <a class="reference internal" href="#streams-developer-guide-dsl-windowing"><span class="std std-ref">windowed</span></a> joins or
                         non-windowed joins.</p>
                     <table border="1" class="docutils">
-                        <colgroup>
```

Comment:
For older version folders, e.g. 10, 0110, etc, do we need to do the same?

Reply:
I see only for 10 - created another PR